common/src/stack/switch/command/list/switch/partition/member/imp_m7800.py

```diff
@@ -0,0 +1,72 @@
+# @copyright@
+# Copyright (c) 2006 - 2019 Teradata
+# All rights reserved. Stacki(r) v5.x stacki.com
+# https://github.com/Teradata/stacki/blob/master/LICENSE.txt
+# @copyright@
+import stack.commands
+from stack import api
+from stack.switch.m7800 import SwitchMellanoxM7800
+
+class Implementation(stack.commands.Implementation):
+	def get_host_interface_by_mac(self, mac):
+		# Return the host name, interface name, and address
+		# if a given mac belongs to a host in the database
+		host = ''
+		interface_name = ''
+		address = ''
+		for interface in api.Call('list host interface'):
+			# If a mac addresses matches what's in the database
+			# Get it's host name and interface name
+			if interface['mac'] and mac.lower() in interface['mac'].lower():
+				found_mac = interface['mac']
+				if 'host' in interface:
+					host = interface['host']
+				if 'ip' in interface:
+					address = interface['ip']
+				if 'interface' in interface:
```

Comment:
These 3 `if` statements could be simplified using `.get("<key>", "")`. This would potentially remove the need for declaring host, address, and interface_name above.

```
host = interface.get("host", "")
address = interface.get("ip", "")
interface_name = interface.get("interface", "")
```